misc/scripts/build.sh

```diff
@@ -140,6 +139,9 @@ else
     -e "ESX=$2" \
     -e "VM1=$3" \
     -e "VM2=$4" \
+    -e "DS1=$DS1" \
+    -e "VM1NAME=$VM1NAME" \
```

Comment:
I'm not sure, each test is doing something of its own for input variables. I saw the govc functions but with my ESX (6.6) keeps failing with "connection reset by peer). I'll use govc once these changes are in and I can verify with 6.5 ESX later.